check_pgactivity

```diff
@@ -500,20 +500,15 @@ sub list_services() {
 
 # Check wrapper around Storable::file_magic to fallback on
 # Storable::read_magic under perl 5.8 and below
+# WARNING: You must hold a lock on the lockfile **BEFORE** calling this sub
```

Comment:
and also mention that caller is responsible for checking that the given $storage exists, isn't a directory and is readable?

Nothing prevents outside modification of the file so an error could still happen, but we should at least be extra clear of what our own API is.